modules/ROOT/pages/compatibility-versions-features.adoc

```diff
@@ -137,14 +137,22 @@ IMPORTANT: With xref:sdk-authentication-overview.adoc[RBAC], in Couchbase Server
 [#jdk-compat]
 == JDK Version Compatibility
 
-The Java SDK is tested with the Oracle JDK which should be used for best experience and supportability.
-Other JDK implementations might work but are  not tested and are unsupported.
+The Java SDK is tested with Oracle JDK and OpenJDK.  Other JDK implementations might work but are not tested and are unsupported.
```

Comment:
For Asciidoc we've adopted the convention of "one sentence per line."

https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
fixed, will update